index.js

```diff
@@ -18,13 +18,13 @@ function calcBudget() {
 }
 
 function blockExtraUser() {
-  if (inputVegan.value > inputPerson.value) {
+  if (Number(inputVegan.value) > Number(inputPerson.value)) {
```

Comment:
Maybe next time we could create new variables for the `.value` addition. For example: `const NumberInputVegetarian = Number(inputVegeterian.value);`